commitizen/git.py

```diff
@@ -98,13 +98,16 @@ def add(args: str = "") -> cmd.Command:
 
 
 def commit(
-    message: str, args: str = "", committer_date: str | None = None
+    message: str,
+    args: str = "",
+    extra_args: str = "",
```

Comment:
Would it be possible for us to combine these 2? Not sure what's the difference between them

Reply:
`extra_args` are arguments that we just want to pass to `git` transparently (like `-S`).

Having the `extra_args` argument separate helps with testing and clarity.

For example:`extra_args` has to start with "--" and contain at least one flag behind it. (e.g. `"-- -S"`). 

If this is contained within the `args` argument, then it becomes a little harder to know whether or not any extra args were passed.

> Would it be possible for us to combine these 2?

`git.commit` is only called with `extra_args` in [commands/commit.py `__call__` method](https://github.com/SCrocky/commitizen/blob/feat-451-add-commit-args/commitizen/commands/commit.py#L108) where it doesn't have any `args`.

So technically, if I'm not missing anything, I could just use the `args` argument to pass the extra_args. 

> Not sure what's the difference between them

I'll add a Docstring

Reply:
> So technically, if I'm not missing anything, I could just use the args argument to pass the extra_args.

Yep, I think that's what we should do. Thanks for clarifying!

Reply:
Done